sonar-dotnet-shared-library/src/test/java/org/sonarsource/dotnet/shared/plugins/FileTypeSensorTest.java

```diff
@@ -82,13 +82,42 @@ public void should_describe() {
   }
 
   @Test
-  public void whenProjectOutPathsNotPresent_shouldNotAddProjectInfo() {
+  public void whenProjectOutPathsNotPresent_shouldNotAddProjectInfo_shouldNotLog() {
     when(settingsMock.getString("sonar.projectName")).thenReturn("FOO_PROJ");
     when(settingsMock.getString("sonar.projectKey")).thenReturn("FOO_PROJ:GUID");
     when(settingsMock.getString("sonar.projectBaseDir")).thenReturn("BASE DIR");
     // note that below it's the singular - Path
-    when(settingsMock.getString("sonar.cs.analyzer.projectOutPath")).thenReturn("CS PATH");
-    when(settingsMock.getString("sonar.vbnet.analyzer.projectOutPath")).thenReturn("VB PATH");
+    when(settingsMock.getString("sonar.cs.analyzer.projectOutPath")).thenReturn("CS OutPath");
+    when(settingsMock.getString("sonar.vbnet.analyzer.projectOutPath")).thenReturn("VB OutPath");
+    // should query the `getStringArray` method, not the `getString` method like below
+    when(settingsMock.getString("sonar.cs.analyzer.projectOutPaths")).thenReturn("CS OutPaths");
```

Comment:
not sure I understand the scenario here. the test name specifies that the `projectOutPaths` is not present but we do specify a value here. Additionally this value is not in the array form as expected.

Reply:
Indeed. I will improve the test name in my next PR in which I add ITs for checking the UI warning is sent to SQ

It's basically a regression test to make sure we use the scanner API correctly. As the `projectOutPaths` is a multi-value property, it should be retrieved via `getStringArray` only.